src/Utils/utils.ts

```diff
@@ -485,3 +485,20 @@ export const mergeQueryOptions = <T extends object>(
     ),
   ];
 };
+
+export const parseOccupationFromExt = (occupation: string) => {
+  switch (occupation) {
+    case "STUDENT":
+      return 1;
+    case "BUSINESSMAN":
+      return 2;
+    case "HEALTH_CARE_WORKER":
+      return 3;
+    case "HEALTH_CARE_LAB_WORKER":
+      return 4;
+    case "ANIMAL_HANDLER":
+      return 5;
+    default:
+      return 6;
+  }
+};
```

Comment:
We are having multiple sources of truth this way.

Refactor this to either do a `OCCUPATION_TYPES.find` like how we do for similar choice fields in care, or use a object map over switch case.